src/clover_leaf.cpp

```diff
@@ -74,11 +74,14 @@ int main(int argc, char *argv[]) {
 	}
 
 
-	std::unique_ptr<global_variables> config = initialise(parallel,
-	                                                      std::vector<std::string>(argv + 1, argv + argc));
+	auto config = initialise(parallel, std::vector<std::string>(argv + 1, argv + argc));
 
 	std::cout << "Launching hydro" << std::endl;
-	hydro(*config, parallel);
+	hydro(config, parallel);
+
+	finalise_field(config);
```

Comment:
This is something new for the OpenMP target version specifically, so worth adding a comment saying what it does.

Reply:
Addressed in https://github.com/UoB-HPC/cloverleaf_openmp_target/pull/2/commits/030edc6cb280ba3f773a581bbd53393e3d9828b3